packages/metal-clay-button/src/MetalClayButton.js

```diff
@@ -61,11 +61,10 @@ MetalClayButton.STATE = {
 	 * @instance
 	 * @memberof MetalClayButton
 	 * @type {?string}
-	 * @default 'default'
+	 * @default undefined
 	 */
 	style: {
 		validator: validators.string,
-		value: 'default'
```

Comment:
Probably will be nice to use the Config object mentioned here: https://github.com/liferay/com-liferay-osb-faro-build-private/wiki/Metal-Soy-Guidelines#declaring-params